include/nn/uds/Cafe.h

```diff
@@ -0,0 +1,471 @@
+    *
+    * \param maxNodes
+    * The maximum amount of nodes that will be able to connect to the network,
+    * including the host. Must be a number between 2 and 9 inclusively.
+    *
+    * \param localCommunicationId
+    * Local communication ID used for identifying the network.
+    *
+    * \param passphrase
+    * Passphrase to be used for the network.
+    *
+    * \param passphraseSize
+    * Size of the passphrase, including NULL termination. Must be greater than 8
+    * characters and lower than 255 characters inclusively.
+    *
+    * \param unused
+    * This parameter is unused.
+    *
+    * \param channel
+    * Channel to use for hosting the network. Use 0 to choose a channel automatically.
+    * Must be either 0, 1, 6 or 11.
+    *
+    * \param appData
+    * Pointer to application data.
+    *
+    * \param appDataSize
+    * Size of application data. Must be lower than 200 bytes inclusively.
+    */
+   nn::Result
+   CreateNetwork(uint8_t id8,
+                 uint8_t maxNodes,
+                 uint32_t localCommunicationId,
+                 const char *passphrase,
+                 uint32_t passphraseSize,
+                 bool unused,
+                 uint8_t channel,
+                 const void *appData,
+                 uint32_t appDataSize) asm("CreateNetwork__Q3_2nn3uds4CafeFUcT1UiPCcT3bT1PCvT3");
+
+   /**
+    * Destroys the given endpoint.
+    *
+    * \param fd
+    * The endpoint descriptor.
+    */
+   nn::Result
+   DestroyEndpoint(EndpointDescriptor *fd) asm("DestroyEndpoint__Q3_2nn3uds4CafeFPQ4_2nn3uds4Cafe18EndpointDescriptor");
+
+   /**
+    * Destroys the previously created network.
+    */
+   nn::Result
+   DestroyNetwork() asm("DestroyNetwork__Q3_2nn3uds4CafeFv");
+
+   /**
+    * Prevents clients from connecting to the network.
+    *
+    * \param unknown
+    * Triggers an unknown flag.
+    */
+   nn::Result
+   DisallowToConnect(bool unknown) asm("DisallowToConnect__Q3_2nn3uds4CafeFb");
+
+   /**
+    * Ejects the specified client from the network.
+    *
+    * \param networkNodeId
+    * The network node ID of the client. Use \link UDS_BROADCAST_NODE_ID \endlink to eject all clients.
+    */
+   nn::Result
+   EjectClient(uint16_t networkNodeId) asm("EjectClient__Q3_2nn3uds4CafeFUs");
+
+   /**
+    * Ejects all spectators from the network.
+    */
+   nn::Result
+   EjectSpectator() asm("EjectSpectator__Q3_2nn3uds4CafeFv");
+
+   /**
+    * Finalizes UDS.
+    */
+   void
+   Finalize() asm("Finalize__Q3_2nn3uds4CafeFv");
+
+   /**
+    * Gets the application data from the network.
+    *
+    * \param appData
+    * Buffer where the application data is written.
+    *
+    * \param readSize
+    * Pointer where the size of the application data is written.
+    *
+    * \param size
+    * Size of the buffer.
+    */
+   nn::Result
+   GetApplicationDataFromBeacon(void *appData, uint32_t *readSize, uint32_t size) asm("GetApplicationDataFromBeacon__Q3_2nn3uds4CafeFPvPUiUi");
```

Comment:
Can you move these parameters into their own lines to be consistent with the other functions in this file?